### PR TITLE
Add test for `@context` redefinition.

### PR DESCRIPTION
**Pull request by davidlehn**
See https://github.com/w3c/json-ld-api/issues/590.

- Please double check this is the correct error code.
- The test naming is still confusing, I think erNN is ok?

### CODE REVIEW

**Review by niklasl — COMMENTED** [review]
Looks good to me! ([TRLD](https://github.com/niklasl/trld) passes these two added tests.)

I just think the HTML versions of the manifests need to be regenerated.

**Review by gkellogg — APPROVED** [review]

**Review by niklasl — APPROVED** [review]

File: tests/toRdf-manifest.jsonld [hunk 1]
```diff
@@ -2231,19 +2231,26 @@
...
+      "@type": [ "jld:NegativeEvaluationTest", "jld:ToRDFTest" ],
+      "name": "Invalid term definition, multiple values for @type.",
+      "purpose": "The value of @type in an expanded term definition object MUST be a string or null.",
+      "input": "toRdf/er55-in.jsonld",
+      "expectErrorCode": "invalid type mapping"
+    }, {
+      "@id": "#ter56",
+      "@type": [ "jld:NegativeEvaluationTest", "jld:ToRDFTest" ],
+      "name": "Invalid redefinition of @context keyword.",
+      "purpose": "Verifies that an exception is raise when attempting to redefine @context.",
```
Comment (dlongley) [inline tests/toRdf-manifest.jsonld:2251]:
```suggestion
      "purpose": "Verifies that an exception is raised when attempting to redefine @context.",
```

**Review by dlongley — APPROVED** [review]

File: tests/expand-manifest.jsonld [hunk 2]
```diff
@@ -1819,6 +1819,13 @@
       "purpose": "The value of @type in an expanded term definition object MUST be a string or null.",
       "input": "expand/er55-in.jsonld",
       "expectErrorCode": "invalid type mapping"
+    }, {
+      "@id": "#ter56",
+      "@type": [ "jld:NegativeEvaluationTest", "jld:ExpandTest" ],
+      "name": "Invalid redefinition of @context keyword.",
+      "purpose": "Verifies that an exception is raise when attempting to redefine @context.",
```
Comment (dlongley) [inline tests/expand-manifest.jsonld:1826]:
```suggestion
      "purpose": "Verifies that an exception is raised when attempting to redefine @context.",
```

**Comment by gkellogg** [comment]
This PR was discussed in a [meeting](https://json-ld.org/minutes/2024-03-06/#25).

<blockquote>
<div onmouseout="$('#link-25').hide()" onmouseover="$('#link-25').show()" id="25" class="information">PR w3c/json-ld-api#591 <a id="link-25" style="display:none;" href="#25">✪</a></div>
<div onmouseout="$('#link-26').hide()" onmouseover="$('#link-26').show()" id="26" class="information"><a href="https://github.com/w3c/json-ld-api/pull/591">https://github.com/w3c/json-ld-api/pull/591</a> -&gt; Pull Request 591 Add test for `@context` redefinition. (by davidlehn) [test:missing-coverage] <a id="link-26" style="display:none;" href="#26">✪</a></div>
<div onmouseout="$('#link-27').hide()" onmouseover="$('#link-27').show()" id="27" class="comment"><span class="name">Niklas Lindstr&#246;m</span>:  Looks good, we can merge. <a id="link-27" style="display:none;" href="#27">✪</a></div>
<//blockquote>

**Pull request merged**